lib/crawly/manager.ex

```diff
@@ -99,7 +99,7 @@ defmodule Crawly.Manager do
 
     maybe_stop_spider_by_itemcount_limit(
       state.name,
-      items_count,
+      delta,
```

Comment:
incorrect, `items_count` should be passed, as we are comparing items scraped, not scrape speed.

Reply:
Good point, fixed